testbed/lib/main.dart

```diff
@@ -38,6 +38,7 @@ void main() {
 
   // Try to resize and reposition the window to be half the width and height
   // of its screen, centered horizontally and shifted up from center.
+  WidgetsFlutterBinding.ensureInitialized();
```

Comment:
This fixes an assertion with trying to use the window size plugin too early. Presumably unrelated, so I could break it out if you prefer, but since it was a trivial fix that I found while making this change I didn't want to stop, switch branches, retest, etc.

Reply:
Verified on macOS and moved to https://github.com/google/flutter-desktop-embedding/pull/497. I'll remove from this PR before landing.